examples/nims/boltz-2/YAML_GUIDE.md

```diff
@@ -381,8 +381,8 @@ The `examples/` directory contains sample YAML files:
 
 ## Further Reading
 
-- [Official Boltz Repository](https://github.com/jwohlwend/boltz)
-- [Boltz-2 Technical Report](https://github.com/jwohlwend/boltz)
+- [Boltz-2 Python Client](https://github.com/NVIDIA/digital-biology-examples/tree/main/examples/nims/boltz-2)
+- [Boltz-2 Technical Report](https://cdn.prod.website-files.com/68404fd075dba49e58331ad9/6842ee1285b9af247ac5a122_boltz2.pdf)
```

Comment:
Should we link to bioRxiv (https://www.biorxiv.org/content/10.1101/2025.06.14.659707v1) instead of this CDN link?